### PR TITLE
Display important copy to warn about Award Reputation

### LINKED ISSUE #3337: Award Reputation does not include the important warning copy.

### PR DESCRIPTION
**Pull request by danbr**
## Description

This PR displays copy to warn about exercising caution when using Award Reputation (not shown on Smite dialog).

Screenshot 2022-05-03 at 19 05 03

I added a few extra minor fixes that were bothering me as i frequently navigated to the dialog. These include:

1- (Award) reputation should be capitalised
2- Smite should be 'Smite Reputation'
3- Within the descriptions capitalisation of 'reputation' should consistent
4-(in Smite dialog) Title currently 'Smite' should be 'Smite Reputation'

Screenshot 2022-05-03 at 17 06 52

Resolves #3337

### CODE REVIEW

File: src/modules/dashboard/components/Dialogs/AwardAndSmiteDialogs/ManageReputationDialogForm/ManageReputationDialogForm.tsx [hunk 1]
```diff
@@ -305,8 +315,31 @@ const ManageReputationDialogForm = ({
...
+                      </ExternalLink>
+                    ),
+                  }}
+                />
+              </p>
+            </div>
+          )}
         </div>
       </DialogSection>
+      {!isSmiteAction && <hr className={styles.divider} />}
```
Comment (rdig) [inline src/modules/dashboard/components/Dialogs/AwardAndSmiteDialogs/ManageReputationDialogForm/ManageReputationDialogForm.tsx:342]: Not a big fan of `hr` elements, but you used it correctly in this instance, so I can't even nit pick it :P

**Review by rdig — APPROVED** [review]
Looks good. Nice job!

**Review by ArmandoGraterol — APPROVED** [review]
Looks good to me :ok_hand:

**Review by chinins — APPROVED** [review]
Good that you've fixed the inconsistencies in the namings & descriptions - nice catch!

**Pull request merged**